packages/neon-api/__tests__/provider/neoscan/core.ts

```diff
@@ -118,6 +118,7 @@ describe("getBalance", () => {
         tokens: {
           TEST: 100,
         },
+        tokenHashes: {},

```

Comment:
Is this possible? Will Neoscan ever return a filled token balance with no hash?

Reply:
correction applied